src/Network/Network/help/Add-AzLoadBalancerBackendAddressPoolTunnelInterface.md

```diff
@@ -0,0 +1,135 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/powershell/module/az.network/add-azLoadBalancerBackendAddressPoolTunnelInterface
+schema: 2.0.0
+---
+
+# Add-AzLoadBalancerBackendAddressPoolTunnelInterface
+
+## SYNOPSIS
+Adds a tunnel interface to a backend address pool of a load balancer.
+
+## SYNTAX
+
+```
+Add-AzLoadBalancerBackendAddressPoolTunnelInterface -BackendAddressPool <PSBackendAddressPool>
+ [-Type <String>] [-Protocol <String>] [-Identifier <UInt32>] [-Port <UInt32>]
+```
+
+## DESCRIPTION
+The **Add-AzLoadBalancerBackendAddressPoolTunnelInterface** cmdlet adds a tunnel interface to a backend address pool configuration of a load balancer..
+
+## EXAMPLES
+
+### Example 1: Add a tunnel interface to a backend address pool configuration to a load balancer
+```powershell
+
+## Get single backend under loadbalancer
+PS C:\> $lb = Get-AzLoadBalancer -ResourceGroupName $resourceGroup -Name $loadBalancerName
+
+## Get all backends under loadbalancer
+PS C:\> $pool = Get-AzLoadBalancerBackendAddressPool -Name "BackendAddressPool02"
+PS C:\> Add-AzLoadBalancerBackendAddressPool -Protocol Vxlan -Type Internal -Port 2000 -Identifier 800 -BackendAddressPool $pool
+PS C:\> Add-AzLoadBalancerBackendAddressPool -Protocol Vxlan -Type External -Port 2001 -Identifier 801 -BackendAddressPool $pool
+```
+If the properties are not provided then they will be replaced with default values.
```

Comment:
Is this okay?

Reply:
If the properties are not provided then they will be replaced with default values of <specify the default values here>